src/rewards/rewards.controller.ts

```diff
@@ -33,4 +41,28 @@ export class RewardController {
   ): Promise<RewardMerkleClaimModel> {
     return this.rewardsService.getUserRewards(Chain.getChain(chain), address);
   }
+
+  @Get('/schedules/vault/:address')
+  @ContentType('json')
+  @Summary('Get all token rewards emmited for vault on network')
+  @Description('Return emission schedule list for specified vault')
+  @Returns(200, RewardSchedulesByVaultModel)
+  @Returns(404).Description('No rewards token emission data for network')
+  @Returns(404).Description('Unknown vault')
+  async getRewardListSchedulesForVault(
+    @PathParams('address') address: string,
+    @QueryParams('chain') chain?: Network,
+  ): Promise<RewardSchedulesByVault> {
+    return this.rewardsService.rewardSchedulesByVault(Chain.getChain(chain), address);
+  }
+
+  @Get('/schedules/vault/list')
```

Comment:
similarly we can host this as just `/schedules` since the list endpoints are generally just at aggregate roots like this